### PR TITLE
Add sheikhs Juhany and Qasim

### LINKED ISSUE #1379: Add new gapless reciter :sheikh Juhaynee
السلام عليكم ورحمة الله وبركاته 
Too many people ask about sheikh Juhaynee.
Now I have timing database file db but we have an issue in our server https://download.quranicaudio.com/quran/abdullaah_3awwaad_al-juhaynee/
 for this sheikh  so I suppose to use other server till repair it.
http://abdullahhussein.com/mushaf_files/sound_sura/abdullah_juhayne_sura/

### LINKED ISSUE #1384: Add new gapless reciter: Sheikh Abdul Muhsin_alqasim
السلام عليكم ورحمة الله وبركاته 
Some people ask for Sheikh Abdul Muhsin_alqasim.
I have Alhamdo lilah the timing database file abdul_muhsin_alqasim.db
The audio files from https://download.quranicaudio.com/quran/abdul_muhsin_alqasim/

Surah Taweba 09 and Yunus 10 from  mp3quran 
https://server8.mp3quran.net/qasm/009.mp3
https://server8.mp3quran.net/qasm/010.mp3

### PR DESCRIPTION
**Pull request by ahmedre**
JazakumAllah khairan brother Nacer. Fixes #1379 and fixes #1384.

### CODE REVIEW

File: common/audio/src/main/res/values-ru/readers.xml [hunk 1]
```diff
@@ -54,4 +54,6 @@
   <string name="qari_ajamy_gapless">Ахмед ибн Али аль-Аджами (безинтервальное)</string>
   <string name="qari_ali_jaber_gapless">Али Джабер (безинтервальное)</string>
   <string name="qari_muaiqly_gapless">Махир аль-Муаккили (безинтервальное)</string>
+ <string name="qari_juhany_gapless">Абдулла Аль-Джахани (безинтервальное)"</string>
```
Comment (ozbek) [inline common/audio/src/main/res/values-ru/readers.xml:57]: For consistency, please use аль instead of Аль

**Comment by ozbek** [comment]
Please remove the double quotes at the end, it is everywhere.

**Comment by ahmedre** [comment]
jazakumAllah khairan, was script translating these 😂

**Comment by nacer80** [comment]
السلام عليكم ورحمة الله وبركاته 
Please correct the arabic name of Sheikh Juhaynee to: 
" عبدالله الجهني "

وجزاكم الله خيرا

**Comment by ahmedre** [comment]
وعليكم السلام ورحمة الله وبركاته
جزاك الله خيراً
done in #1403 
والسلام عليكم

**Pull request merged**